_posts/2017-08-05-MusED-Hackathon-2017.md

```diff
@@ -0,0 +1,21 @@
+---
+title:  "MusED Hackathon 2017"
+date:   2017-09-04 12:00:00 -0400
+categories: update
+---
+The [MusED Hackathon]({{site.baseurl}}{% link _events/2017-08-05.md %}), held on August 5, revolved around Music Education, Music Experience Design, and musing about and muses for Music. It was held in collaboration with the [IMPACT Conference](https://impact.musedlab.org/) and NYU Steinhardt's [MusEDLab](https://musedlab.org/) lab.
+
+After kicking off the day with a set of invigorating talks at 12 pm and a pizza break before 2 pm, participants took part in workshops or gathered in groups and created hacks.
+
+![Img: Thoughts from the talks](/assets/events/20170805/20170805_talk4.jpg)
```

Comment:
Is it standard to do "Img: "?

The alt text is for accessibility, it generally describes the picture. "Thoughts from the talks" might leave the user wanting more.

Reply:
I don't know the consensus about doing it in Markdown. Just that it's standard in TeX. It didn't render at first when  I didn't have the Img: but now it's fine, so perhaps something else went wrong and I inadvertently fixed it.